netlogo-gui/src/main/properties/MetricsBooleanEditor.scala

```diff
@@ -0,0 +1,21 @@
+package org.nlogo.properties
+
+import scala.collection.mutable.ArrayBuffer
+
+class MetricsBooleanEditor(accessor: PropertyAccessor[Boolean], properties: ArrayBuffer[PropertyEditor[_]]) extends BooleanEditor(accessor)
+{
+    override def changed(): Unit =
+    {
+        properties.find(_.accessor.accessString == "runMetricsCombine") match
```

Comment:
Sorry, I missed this the first go-through.  This `runMetricsCombine` appears to be a leftover from prior work and isn't referenced anywhere else.  I think this code can be removed.

But looking at this class again, I think it'd be preferable that it not even know which property it's meant to represent.  Could it can take in a single `ProperyEditor[_]` at creation so it doesn't have to know about the `accessString` at all?  I don't consider that necessary to merge, but I would like it looked at at some point.